docs/advanced_reference/commands/autonomy_mint.md

```diff
@@ -0,0 +1,126 @@
+# `autonomy mint`
+
+Tools to mint components on chain
+
+### Options
+
+`--use-ethereum`
+: Use `ethereum` to chain profile to interact with the contracts.
+
+`--use-goerli`
+: Use `goerli` to chain profile to interact with the contracts.
+
+`--use-custom-chain`
+: Use custom-chain to chain profile to interact with the contracts.
+
+`--use-local`
+: Use local to chain profile to interact with the contracts.
+
+`--skip-hash-check`
+: Skip hash check when verifying dependencies on chain
+
+### Chain selection
+
+This command group contains tools for minting components, agents and service packages. You can specify what type of chain you want by providing a `--use-CHAINTYPE` flag. The current implementation provides 4 different types of chain to choose from `local`, `goerli`, `ethereum` and `custom` chain. 
+
+If you use the local chain, make sure you have a local hardhat deployment with the required registry contracts deployed. If you want to use a custom chain you'll have to provide the chain parameters as the environment variables.
+
+Required variables if you want to use a custom chain
+
+- `CUSTOM_COMPONENT_REGISTRY_ADDRESS` : custom `component_registry` contract address
+- `CUSTOM_AGENT_REGISTRY_ADDRESS` : custom `agent_registry` contract address
+- `CUSTOM_REGISTRIES_MANAGER_ADDRESS` : custom `registries_manager` contract address
+- `CUSTOM_SERVICE_MANAGER_ADDRESS` : custom `service_manager` contract address
+- `CUSTOM_SERVICE_REGISTRY_ADDRESS` : custom `service_registry` contract address
+- `CUSTOM_GNOSIS_SAFE_MULTISIG_ADDRESS` : custom `gnosis_safe_multisig` contract address
+- `CUSTOM_CHAIN_RPC` : RPC address
+- `CUSTOM_CHAIN_ID` : Chain ID
+
+In the same way, if you want to use chain other than custom and local you'll have to provide the RPC address as following environment variables.
+
+- `GOERLI_CHAIN_RPC` : RPC endpoint for `goerli` testnet
+- `ETHEREUM_CHAIN_RPC` : RPC endpoint for `ethereum` mainnet
+
+## `autonomy mint protocol/contract/connection/skill`
+
+### Usage
+
+`autonomy mint COMPONENT_TYPE [OPTIONS] PACKAGE_PATH KEYS`
+
+### Options
+
+`--password TEXT`
+: Password for key pair
+
+`-d, --dependencies TEXT`
+: Dependency for the package
+
+`--nft TEXT`
```

Comment:
It is a bit strange that the name of this parameter is "nft" and then it requires a hash as argument. (I would call it --nft-hash for example)